rfcs/build-system/05-folder-organization.md

```diff
@@ -0,0 +1,65 @@
+# RFC: Fluent UI Repo Folder Organization
+
+---
+
+Contributors: Fluent UI React Build team, @justSlone
+
+## Summary
+
+This propsoal covers 2 topics:
+1. organization of rfcs folder
+2. organization (or removal) of specs folder
+
+Overall this RFC proposes that we organize our repo by NPM package names under the @fluentui namespace. 
+These names today are `react`, `react-components`, `react-northstar`, and `web-components`. Organizing by NPM package name, while not necessarily ideal and may change over time, has the benefit of making it easy to follow from NPM to the appropriate source code in GitHub. This also scales well if we add additional projects to the repo (as long as they are under the @fluentui namespace on NPM). 
+
+## Background
+
+Some useful background is that we have heard from contributors that the current organization of the rfcs and specs folders at the root of the 
+repo is confusing to contributors working on web-components. The RFCs and Specs don't mention clearly what project they are for. 
+
+## Problem statement
+
+With the addition of vNext to the repo we now have 4 actual projects in our repo: 
+`@fluentui/react`, `@fluentui/react-northstar`, `@fluentui/web-components`, and more recently `@fluentui/react-components`.
+
+This leads to a fairly confusing and difficult to navigate repo. In particular we have some folders we are actively using, such as rfcs and specs which could apply to any of the 4 projects, but in fact only really apply to @fluentui/react-components. This is potentially confusing to contributors visiting our repo, and generally a ramp-up headache. 
+
+## Detailed Design or Proposal
+
+1. Within the rfcs folder we should create a new subfolder called `react-components` and move all existing rfcs to this folder. 
+   - if needed additional `react`, `react-northstar`, or `web-components` RFCs can be created
+   - RFCs that apply to the whole repo can go in a folder called `shared` 
```

Comment:
Just a thought, but I wonder if we should break this down by technology rather than broad subfolder, that seems like it could scale better in the future without requiring another move and it may work better when the repo reorganization takes place:

```
> react
    > react-components
    > react-northstar
    > ...
> web-components
```

Reply:
To me this feels like unnecessary nesting. I don't expect we will mix and match so many technologies within one monorepo for this to provide a considerable impact.

Reply:
Yeah, let's keep it without the additional nesting. If at some point we decided to fold in react native and other platforms into the repo I think it would make sense to add more nesting. Right now we don't have plans to do that though.